datafusion/sqllogictest/test_files/tpch/answers/q6.slt.part

```diff
@@ -27,4 +27,4 @@ where
   and l_discount between 0.06 - 0.01 and 0.06 + 0.01
   and l_quantity < 24;
 ----
-11803420.2534
+7115406.7008
```

Comment:
AFAICT this result changes because the literals are floats, and `l_discount` is a decimal. Is this a correctness problem? @andygrove

Reply:
SQL literals should be decimal not float per ANSI SQL specification

Reply:
Yes, this is a correctness problem. I explained in https://github.com/apache/datafusion/pull/14273#discussion_r1936007682